src/test/java/de/agilecoders/wicket/demo/PageB.html

```diff
@@ -7,9 +7,14 @@
     Wicket - RequireJs integration 2
 
     <script type="text/javascript">
-        require( ['wicket!pageB'], function(pageB) {
+//         require( ['wicket!pageB'], function(pageB) {
+//             pageB.b();
+//         });
```

Comment:
As I mentioned at #9 it is better to remove unused code instead of leaving it commented out.